components/uniset/src/uniset.rs

```diff
@@ -289,12 +324,14 @@ mod tests {
         assert_eq!(ex, check.inv_list);
         assert_eq!(2, check.size());
     }
+
     #[test]
     fn test_unicodeset_try_from_vec_error() {
         let check = vec![1, 1, 2, 3, 4];
         let set = UnicodeSet::from_inversion_list(check.clone());
-        assert_eq!(Err(UnicodeSetError::InvalidSet(check)), set);
+        assert!(matches!(set, Err(UnicodeSetError::InvalidSet(set))));
```

Comment:
Whoops.  `matches!` only works on const expressions.  You need to split the equals comparison into its own block.

```suggestion
        assert!(matches!(set, Err(UnicodeSetError::InvalidSet(_))));
        if let Err(UnicodeSetError::InvalidSet(actual)) = set {
            assert_eq!(check, actual);
        }
```